src/axom/inlet/Table.cpp

```diff
@@ -145,25 +144,25 @@ std::shared_ptr<Table> Table::addStringArray(const std::string& name,
   return table;
 }
 
-bool Table::getBoolArray(std::unordered_map<int, bool>& map)
+bool Table::getArray(std::unordered_map<int, bool>& map)
 {
   return axom::utilities::string::endsWith(m_name, "_inlet_array") &&
     m_reader->getBoolMap(m_name.substr(0, m_name.size() - 12), map);
 }
 
-bool Table::getIntArray(std::unordered_map<int, int>& map)
+bool Table::getArray(std::unordered_map<int, int>& map)
 {
   return axom::utilities::string::endsWith(m_name, "_inlet_array") &&
     m_reader->getIntMap(m_name.substr(0, m_name.size() - 12), map);
 }
 
-bool Table::getDoubleArray(std::unordered_map<int, double>& map)
+bool Table::getArray(std::unordered_map<int, double>& map)
 {
   return axom::utilities::string::endsWith(m_name, "_inlet_array") &&
     m_reader->getDoubleMap(m_name.substr(0, m_name.size() - 12), map);
 }
 
-bool Table::getStringArray(std::unordered_map<int, std::string>& map)
+bool Table::getArray(std::unordered_map<int, std::string>& map)
 {
   return axom::utilities::string::endsWith(m_name, "_inlet_array") &&
```

Comment:
Outside the scope of this PR, but this logic should be centralized into a utility function:
```cpp
  return axom::utilities::string::endsWith(m_name, "_inlet_array") &&
    m_reader->getDoubleMap(m_name.substr(0, m_name.size() - 12), map);
```

Reply:
@joshessman-llnl  also pointed out this was double reading from Lua.. this info is already inside of the datastore.  This was missed in the previous PR.